bddisasm/bdx86_decoder.c

```diff
@@ -98,11 +97,18 @@ NdFetchData(
     ND_UINT8 Size
     )
 {
-    return (4 == Size) ? ND_FETCH_32(Buffer) :
-           (1 == Size) ? ND_FETCH_8(Buffer) :
-           (8 == Size) ? ND_FETCH_64(Buffer) :
-           (2 == Size) ? ND_FETCH_16(Buffer) :
-           0;
+    switch (Size) {
```

Comment:
The switch - case statement should adhere to the coding style used throughout the rest of the project:
```
switch (expression)
{
case value1:
    ...
case value2:
    ...
default:
    ...
}
```

Reply:
So i should call it expression instead of Size?

Reply:
No, you should format the code properly, like this:
```
switch (Size) 
{
case 1:
    return ND_FETCH_8(Buffer);
case 2:
    return ND_FETCH_16(Buffer);
case 4:
    return ND_FETCH_32(Buffer);
case 8:
    return ND_FETCH_64(Buffer);
default:
    return 0;
}
```

Reply:
👍